### DISCUSSION TITLE
Are there any screenshots of Tailpress anywhere?

### DISCUSSION THREAD

**Discussion by numbrz1111** [tailpress/tailpress#61] [Q&A] [answered]
Very intrigued by this, may set up on my store, but firstly does anybody have any screenshots of how this looks, or can look, on a wordpress site?

**Comment by jeffreyvr** [comment, marked as answer]
TailPress is a boilerplate - a starting point for a custom WordPress theme you'll develop (with Tailwind CSS). So TailPress itself really doesn't have a particular look.

Here are some video's that give you an indication of what you get started with: https://www.youtube.com/playlist?list=PL6GBdOp044SHIOSCZejodwr1HcYsC43wG

That said, if anyone wants to share a theme/site built with TailPress - feel free to do so of course! :)
